apps/condo/domains/miniapp/schema/B2BAppPermission.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Generated by `createschema miniapp.B2BAppPermission 'app:Relationship:B2BApp:PROTECT; key:Text'`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const get = require('lodash/get')
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { ROLE_PERMISSION_REGEX } = require('@condo/domains/common/constants/regexps')
+const access = require('@condo/domains/miniapp/access/B2BAppPermission')
+const { PERMISSION_KEY_WRONG_FORMAT_ERROR } = require('@condo/domains/miniapp/constants')
+const { updateB2BAppRolesPermissions } = require('@condo/domains/miniapp/tasks')
+
+const B2BAppPermission = new GQLListSchema('B2BAppPermission', {
+    schemaDoc: 'B2BApp permissions that describe additional capabilities within the mini-application ' +
+        'and allow the organization administration to manage accesses within the miniapp ' +
+        'depending on the employee\'s role ' +
+        'by toggling the B2BAppPermissions of a particular application for each role. ' +
+        'By default, for all employees with "canManageIntegrations" in their role, all permissions will be set to True. ' +
+        'For all other roles with access to the application the default value is False.',
+    fields: {
+        app: {
+            schemaDoc: 'Link to the application to which the permission belongs',
+            type: Relationship,
```

Comment:
To keep ks-v5-to-v6 migration
```suggestion
            type: 'Relationship',
```

Reply:
done 👍